src/api/app/controllers/webui/users_controller.rb

```diff
@@ -59,9 +60,8 @@ def create
       redirect_to users_path
     else
       session[:login] = create_params[:login]
-      User.session = User.find_by!(login: session[:login])
-      if User.session.home_project
-        redirect_to project_show_path(User.session.home_project)
+      if Project.find_by(name: "home:#{session[:login]}")
+        redirect_to project_show_path("home:#{session[:login]}")
```

Comment:
```suggestion
      if Project.find_by(name: "home:#{params[:login]}")
        redirect_to project_show_path("home:#{params[:login]}")
```
Not related to your changes, I see the `session[:login] = create_params[:login]` assignment on line 62 unnecessary (I couldn't add this line into the suggestion).

From my point of view, we could simply use `params[:login]` instead of `session[:login]`. Maybe a refactoring for later, after this PR...

Reply:
If you don't assign `session[:login] ` then the person signing up isn't signed in afterward.